### PR TITLE
[T2A5][T09-A3]NgHuanRan

### PR DESCRIPTION
**Pull request by shusiner**
ready for review

### CODE REVIEW

**Comment by jeffryhartanto** [comment]
Please mention your Team ID, e.g [T09-A1]. Make sure you use the correct title next time.

**Comment by shusiner** [comment]
ok i did this when i didnt know my group

**Comment by mauris** [comment]
@shusiner please use a dash instead of underscore. `[T09-A3]` instead of `[T09_A3]`

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -420,7 +420,7 @@ private static String getMessageForSuccessfulAddPerson(String[] addedPerson) {
      * @return feedback display message for the operation result
      */
     private static String executeFindPersons(String commandArgs) {
-        final Set<String> keywords = extractKeywordsFromFindPersonArgs(commandArgs);
+        final Set<String> keywords = extractKeywordsFromFindPersonArgs(commandArgs.toLowerCase());
```
Comment (z11i) [inline src/seedu/addressbook/AddressBook.java:423]: Remember to include this in the comments so that it's clear that commands are case insensitive.

**Pull request closed without merging**